resources/views/livewire/module-content.blade.php

```diff
@@ -0,0 +1,28 @@
+<x-slot name="title">
+    {{ __('Module') . '-' . $module->title }}
```

Comment:
```suggestion
    {{ __('Module - :title', ['title' => $module->title]) }}
```

Please review all string concatenations throughout.